docs/README.md

```diff
@@ -4,5 +4,6 @@ This folder contains advanced guides for `clasp`.
 
 - [Develop clasp](/docs/develop.md): Build, Run, and Test the CLI.
 - [Run](/docs/run.md): Execute Apps Script remotely with `clasp run`.
+- [Running loaclly](/docs/running-locally.md): Install and use clasp as a local dependency.
```

Comment:
There is a typo here

Reply:
Thanks. I usines and merged a PR fixing it.